bioimageio/core/prediction_pipeline/_combined_processing.py

```diff
@@ -1,54 +1,30 @@
+import typing
 import warnings
 from collections import defaultdict
-from typing import Any, Dict, List, Optional, Sequence, Set, Tuple, Type
+from typing import DefaultDict, Dict, List, Optional, Sequence, Set, Tuple
 
 import xarray as xr
 
+from bioimageio.core.measure_groups import MeanVarStd, PercentileGroup
 from bioimageio.core.resource_io import nodes
-from bioimageio.core.statistical_measures import Mean, Measure, Percentile, Std
-from bioimageio.spec.model.raw_nodes import PostprocessingName, PreprocessingName
+from bioimageio.core.statistical_measures import Mean, Measure, MeasureValue, Percentile, Std, Var
+from bioimageio.core.utils import TensorName
 from ._processing import (
-    Binarize,
-    Clip,
+    DatasetMode,
     EnsureDtype,
+    KNOWN_POSTPROCESSING,
+    KNOWN_PREPROCESSING,
+    PER_DATASET,
+    PER_SAMPLE,
     Processing,
-    ScaleLinear,
-    ScaleMeanVariance,
-    ScaleRange,
-    Sigmoid,
-    ZeroMeanUnitVariance,
+    SampleMode,
 )
 
 try:
     from typing import Literal
 except ImportError:
     from typing_extensions import Literal  # type: ignore
 
-KNOWN_PREPROCESSING: Dict[PreprocessingName, Type[Processing]] = {
-    "binarize": Binarize,
-    "clip": Clip,
-    "scale_linear": ScaleLinear,
-    "scale_range": ScaleRange,
-    "sigmoid": Sigmoid,
-    "zero_mean_unit_variance": ZeroMeanUnitVariance,
-}
-
-KNOWN_POSTPROCESSING: Dict[PostprocessingName, Type[Processing]] = {
-    "binarize": Binarize,
-    "clip": Clip,
-    "scale_linear": ScaleLinear,
-    "scale_mean_variance": ScaleMeanVariance,
-    "scale_range": ScaleRange,
-    "sigmoid": Sigmoid,
-    "zero_mean_unit_variance": ZeroMeanUnitVariance,
-}
-
-
-Scope = Literal["sample", "dataset"]
-SAMPLE: Literal["sample"] = "sample"
-DATASET: Literal["dataset"] = "dataset"
-SCOPES: Set[Scope] = {SAMPLE, DATASET}
```

Comment:
What happened to all the stuff that was removed here, was it moved somewhere else (if so I couldn't find that in the diffs), or is it irrelevant now.